cmd/fluxctl/install_cmd.go

```diff
@@ -33,21 +33,23 @@ fluxctl install --git-url 'git@github.com:<your username>/flux-get-started' | ku
 	cmd.Flags().StringVar(&opts.GitBranch, "git-branch", "master",
 		"Git branch to be used by Flux")
 	cmd.Flags().StringSliceVar(&opts.GitPaths, "git-paths", []string{},
-		"Relative paths within the Git repo for Flux to locate Kubernetes manifests")
+		"relative paths within the Git repo for Flux to locate Kubernetes manifests")
```

Comment:
How come these all got lowercased, _en passant_ .. looking at the other subcommands, the usage messages tend to start with capitals

Reply:
Ops, I did it after `fluxd`'s flags (wrongly assuming that the rest of the `fluxctl` commands followed the as pattern)

Reply:
I am happy to revert that, but I think `fluxd` and `fluxctl` should be consistent

Reply:
I noticed it because it caused a merge conflict (easily solved, I just need to know which direction to solve it in ..)

Good point on being consistent with fluxd; I'll resolve the conflict with the least change, then do them all at once in another PR. (i.e., don't worry about reverting!)

Reply:
Great, thanks!